doc/source/whatsnew/v0.20.0.txt

```diff
@@ -398,7 +398,32 @@ To convert a ``SparseDataFrame`` back to sparse SciPy matrix in COO format, you
 
    sdf.to_coo()
 
-.. _whatsnew_0200.enhancements.intervalindex:
+.. _whatsnew_0200.enhancements.style_excel:
+
```

Comment:
you can add a pointer to this section in the top-level highlites (follow the same style there)